core/services/ocr2/plugins/ocr2keeper/evm21/logprovider/provider_life_cycle.go

```diff
@@ -126,7 +126,7 @@ func (p *logEventProvider) register(ctx context.Context, lpFilter logpoller.Filt
 	}
 	// TODO: Optimise to do backfill from ufilter.configUpdateBlock only for new filters
 	// if it is not too old
-	p.poller.ReplayAsync(backfillBlock)
+	// p.poller.ReplayAsync(backfillBlock)
```

Comment:
ignore for now, will be removed after https://github.com/smartcontractkit/chainlink/pull/10470 is merged